docs/sources/explore/logs-integration.md

```diff
@@ -29,26 +29,29 @@ For logs where a **level** label is specified, we use the value of the label to
 
 **Supported log levels and mapping of log level abbreviation and expressions:**
 
-
-|  Supported expressions      | Log level     | Color       |
-| --------------------------- |:-------------:| -----------:|
-| emerg                       | critical      | purple      |
-| fatal                       | critical      | purple      |
-| alert                       | critical      | purple      |
-| crit                        | critical      | purple      |
-| critical                    | critical      | purple      |
-| err                         | error         | red         |
-| eror                        | error         | red         |
-| error                       | error         | red         |
-| warn                        | warning       | yellow      |
-| warning                     | warning       | yellow      |
-| info                        | info          | green       |
-| information                 | info          | green       |
-| notice                      | info          | green       |
-| dbug                        | debug         | blue        |
-| debug                       | debug         | blue        |
-| trace                       | trace         | light blue  |
-| *                           | unknown       | grey        |
+| Supported expressions | Log level |      Color |
```

Comment:
No change to this table - it just auto re-styled after I saved changes based on our current linting.